docs/data/destinations/planhat-event-streaming.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Planhat Event Streaming
+description: Stream Amplitude events to Planhat.
+status: new
+---
+
+!!!beta "This feature is in open beta"
+
+    This feature is in open beta and is in active development. Contact the [Planhat support team]([https://userlist.com/](https://www.google.com/url?q=https://support.planhat.com/en/&sa=D&source=docs&ust=1681362728518389&usg=AOvVaw0HZBKn1FHVmfkZ7_tigFKN)) for support with this integration.
+
+[Planhat](https://www.planhat.com/) is a customer platform built to give insights, manage workflow and drive customer experience. Planhat is helping hundreds of modern technology companies worldwide center their business around their customers in order to maximize customer success and customer lifetime value. 
+
+
+## Considerations
+
+Amplitude Data’s Planhat integration allows you to send a continuous, real-time stream of events as User Activities on the End User, Asset or Project level.
+
+Keep these things in mind when sending events to [Planhat](https://support.planhat.com/en/articles/7181975-setting-up-the-amplitude-integration#h_87f475e94b):
+- You must enable this integration in each Amplitude project you want to use it in.
+- You need to be an admin of a paid Planhat plan to enable this integration.
+- Amplitude matches the **user_id** to the End User **externalId** within Planhat for associated events. If a user with that **externalId** doesn't exist within Planhat, a user with that **externalId** is created. Make sure that the Amplitude **user_id** field matches the Planhat user **externalId** field to avoid user duplication.
+- Relevant limits for Planhat events are:
+  - Maximum size of event data: 1 MB
+- Amplitude sends all user, event, and group properties along with the event.
+- Note that because the Planhat Analytics endpoint is a “data sink”, it will always return status 200 (OK) even when there is an error in the payload. See Planhat’s [API documentation](https://docs.planhat.com/#response_codes) for details.
+
+## Setup
+See the [Planhat documentation](https://www.google.com/url?q=http://support.planhat.com/en/articles/7181975-setting-up-the-amplitude-integration%23h_87f475e94b&sa=D&source=docs&ust=1681362614710965&usg=AOvVaw2y7G6XhmQHzvEoK8v9w8fU) for more details.
```

Comment:
Update the url with: https://support.planhat.com/en/articles/7181975-setting-up-the-amplitude-integration#h_87f475e94b